Development.xcconfig

```diff
@@ -1 +1 @@
-OTHER_SWIFT_FLAGS=$(inherited) -Xfrontend -warn-long-expression-type-checking=80
+OTHER_SWIFT_FLAGS=$(inherited) -Xfrontend -warn-long-expression-type-checking=400
```

Comment:
bumping to what we use in pointfreeco

Reply:
I think everything here is 100ms or under. Would love to not let 400ms stuff sneak in if we can avoid it.

Reply:
I've got about 20 warnings that are over 100ms :/ it's prob all so dependent on the current state of your computer, but right now pretty much everything is definitely less than 400ms